src/ui/widgets/config/forms/NavigationBarConfigForm.js

```diff
@@ -61,8 +50,8 @@ export default class NavigationBarConfigForm extends PureComponent {
       initialValues,
       onSpecificPageChoose,
       appTourProgress,
+      mode,
```

Comment:
have u tested this changes @chalagashvili ?

As far as I know, `mode` is not injected on NavigationBarConfigForm by default(I tried to inject it via props, but it is loaded dynamically by a method and it didn't work).